.github/workflows/tag-new-release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release a new Armeria version
+on:
+  workflow_dispatch:
```

Comment:
I'm not really familiar with this. So is there a UI where we can supply/trigger these inputs?

Reply:
Right. If you click the workflow in https://github.com/line/armeria/actions, we can see the input from the right upper corner.
<img width="431" alt="image" src="https://github.com/line/armeria/assets/1866157/6c9ff335-f251-4750-a47e-d90ab9c2f030">